packages/ui/src/kit/VDateInput/VDateInput.theme-default.scss

```diff
@@ -0,0 +1,130 @@
+.v-date-input.theme-default {
+  position: relative;
+}
+
+.v-date-input.theme-default .v-date-input-field {
+  position: relative;
+}
+
+.v-date-input.theme-default .v-date-input-field-icon {
+  position: absolute;
+  width: 16px;
+  height: 16px;
+  top: 50%;
+  right: 16px;
+  transform: translateY(-50%);
+  pointer-events: none;
+}
+
+.v-date-input.theme-default .v-date-input-calendar {
+  position: absolute;
+  z-index: 1;
+  padding: 8px 4px;
+  border-radius: 4px;
+  background: #fff;
+  box-shadow: 0px 2px 6px rgba(0, 0, 0, 0.3);
```

Comment:
maybe move shadow color to custom vars?

Reply:
Shadows can be different unlike colors and typography

Reply:
my fault, i started searching `box-shadow` and found, that we used `rgba(36, 42, 46, 0.1)` in most of box-shadows colors.
i mean move only `rgba(36, 42, 46, 0.1)` to vars, not this `rgba(0, 0, 0, 0.3)` color.
sorry